documentation/src/main/asciidoc/core/manual/en_US/18_jpql_functions.adoc

```diff
@@ -167,7 +167,8 @@ There are multiple different cast functions for different data types.
 * `java.sql.Timestamp` - `CAST_TIMESTAMP`
 * `java.util.Calendar` - `CAST_CALENDAR`
 
-A cast invocation will always generate a ANSI SQL cast. The SQL data type for a Java type is determined by link:{core_jdoc}/persistence/spi/DbmsDialect.html#getSqlType(java.lang.Class)[`DbmsDialect.getSqlType()`].
+A cast invocation will always generate a ANSI SQL cast. The SQL data type for a Java type is determined by link:{core_jdoc}/persistence/spi/DbmsDialect.html#getSqlType(java.lang.Class)[`DbmsDialect.getSqlType()`]
+and can be overridden using the optional `sqlCastTypeOverride` parameter.
```

Comment:
Please note that the sqlCastTypeOverride is a string and the contents represent the SQL type. Give an example like e.g. `varchar(100)`